src/layouts/page/page.tsx

```
@@ -0,0 +1,36 @@
+        <Trans
+          i18nKey="page.access-denied"
+          components={[<a href="/#!/access-level" key="access-level" />]}
+        />
+      </Alert>
+    )}
+
+    {hasRights && (
+      isLoading ? <Loader className="page-loader" /> : children
+    )}
+
+    <div id="floating-container"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)